raiden/network/transport/matrix/transport.py

```diff
@@ -985,14 +992,45 @@ def _get_room_for_address(self, address: Address) -> Optional[Room]:
             while room_ids:
                 room_id = room_ids.pop(0)
                 room = self._client.rooms[room_id]
-                if not self._is_broadcast_room(room):
-                    self.log.debug("Existing room", room=room, members=room.get_joined_members())
-                    return room
-                self.log.warning(
-                    "Ignoring broadcast room for peer",
+                if self._is_broadcast_room(room):
+                    self.log.warning(
+                        "Ignoring broadcast room for peer",
+                        room=room,
+                        peer=to_normalized_address(address),
+                    )
+                room_candidates.append(room)
```

Comment:
The log message says we're ignoring the broadcast room, but here we add it anyways.

Reply:
🙈 good catch, thanks.

The condition is probably not strictly necessary since `_get_room_ids_for_address` already filters out public rooms (and the only remaining public rooms are the broadcasting ones) but needs to be fixed of course.